crates/rooch-types/src/address.rs

```diff
@@ -41,24 +41,21 @@ pub trait RoochSupportedAddress:
 }
 
 /// Multi chain address representation
-/// The address is distinguished by the coin id, coin id standard is defined in [slip-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)
+/// The address is distinguished by the coin type, coin type standard is defined in [slip-0044](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)
 #[derive(Debug, Clone, PartialEq, Eq, Hash)]
 #[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
 pub struct MultiChainAddress {
-    pub coin_id: CoinID,
+    pub coin: Coin,
```

Comment:
It will conflict with the `Coin` in the contract, when it is necessary to implement the type in Rust

Reply:
> It will conflict with the `Coin` in the contract, when it is necessary to implement the type in Rust

So `CoinType` the name would be good for this scenario?

Reply:
`CoinType` is also used in the contract, `CoinID` has not used yet

Reply:
> CoinType is also used in the contract, CoinID has not used yet

I changed `CoinID` to `Symbol` per https://github.com/satoshilabs/slips/blob/master/slip-0044.md.

We need to figure out a name for full name of each blockchain in the Coin column.

Reply:
I may use `CoinID` to substitute `Coin`.

Reply:
The struct layout of MultiChainAddress in Rust must be the same as the MultiChainAddress in Move.